db/migrate/20201022154530_add_flag_column.rb

```diff
@@ -0,0 +1,16 @@
+class AddFlagColumn < ActiveRecord::Migration[5.2]
+  def change
+    [:evidence_items, :variants, :assertions, :variant_groups, :genes].each do |table|
+      add_column table, :flagged, :boolean, null: false, index: true, default: false
+    end
+
+    [EvidenceItem, Variant, Assertion, VariantGroup, Gene].each do |klass|
```

Comment:
I don't think this will work on a rollback so you'll want to split this out in an `up` and `down` method.

Reply:
Good catch!